api_v3/services/CategoryService.php

```diff
@@ -180,8 +180,6 @@ function deleteAction($id, $moveEntriesToParentCategory = KalturaNullableBoolean
 				$categoryDb->setDeletedAt(time());
 			else 
 				$categoryDb->setDeletedAt(time(), 0);
-				
-			$categoryDb->save();	
```

Comment:
We don't trust the setter to do the change, always save after calling the setter and make sure that nothing is done before the save.